Chore/refactor into multiple files

matyasfodor opened the pull request.

sgalkina commented on model/handlers.py:
```diff
@@ -0,0 +1,111 @@
+
+
+async def map(request):
+    # FIXME: A malicious user can access any JSON file in the system this way.
```
A valid concern, though still only in the map folder?

matyasfodor replied:
No, actually `../..` would allow to access an arbitrary json file. It was added by Moritz 4 months ago, I just moved it. Here's a fix for this: https://stackoverflow.com/a/45188896/2419215 We just have to make sure, that the assembled path is under the safe path (`MAPS_DIR` in this case). For assempling paths it's better to use `os.path.join`, it will handle the `/` signs nicely

matyasfodor replied:
Fixed it and handled most cases with the appropriate HTTP status code. There's a JSON parsing which is unnecessary, but I could not figure out how to do it properly.

Midnighter replied:
There's also `werkzeug.secure_path` that prevents you from breaking out of the current directory I think.

sgalkina commented on model/utils.py:
```diff
@@ -0,0 +1,28 @@
+        return result
+    return wrap
+
+def map_reactions_list(map_path):
```
I'm not sure it belongs to `utils`, maybe not general enough?

matyasfodor replied:
Good point, I moved this to response

matyasfodor commented:
@Midnighter 

- I renamed module_operations, and also `memory_storage` (to `storage` - the redis related code is also there)
- renamed consts to constants
- Fixed logging, thanks for the feedback
- Broke down unit_tests into a separate folder to multiple files.

Please note that this is a PR targeting my other branch with https://github.com/DD-DeCaF/model/pull/61.

Midnighter commented:
Great work! Looks really good now.
